docs/security/email-link-protection.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Email Link Protection
+description: Protect email link sign-ins and sign-ups
+---
+
+# Email Link Protection
```

Comment:
Is **Email Link Protection** going to be the name of the feature and the exact name using in the Dashboard? And will it be fully capitalized in the Dashboard as well?

If so, please use [sentence-casing](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#15-use-sentence-case-for-titles) for headings!

```suggestion
# Email link protection
```

Reply:
This naming isn't going to appear directly in the dashboard, here's a preview of what the UI will look like (it's awkwardly in two places, but this is the best we can do now without a larger dashboard page refactor):


https://github.com/clerk/clerk-docs/assets/27433835/e0fe431d-a1be-4d5b-a278-c0084b9ffd5e